src/main/js/ContentHitRenderer.tsx

```diff
@@ -57,9 +58,14 @@ type SyntaxHighlighting = {
 
 const ContentMessage: FC = ({ children }) => <pre className="p-4 is-size-7 is-family-primary">{children}</pre>;
 
-const BinaryContent: FC = () => {
+const BinaryContent: FC<HitProps> = ({ hit }) => {
   const [t] = useTranslation("plugins");
-  return <ContentMessage>{t("scm-content-search-plugin.hit.binary")}</ContentMessage>;
+
+  return (
+    <ExtensionPoint name="search.content.binary" props={{ hit }}>
```

Comment:
Note to myself: we really have to fix our ExtensionPoint typings again and figure out if and how we share extension points between plugins